OpenUpSA/wazimap-ng-ui
# Added GUI tests for profile default filters and profile view filters

## Description
[Added GUI tests for profile default filters and profile view filters

## Related Issue
https://wazimap.atlassian.net/browse/WNCM-723

## How is it tested automatically?

## How should a reviewer test it locally
Add both default flters and profile default for same group
you will notice that snack bar is not shown in this case

## Screenshots


## Changelog

### Added

### Updated

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
